config/src/main/resources/default-execution-parameters.json

```diff
@@ -5,5 +5,5 @@
   "proto": false,
   "url": null,
   "zipinput": null,
-  "exclude": null
+  "exclude": ""
```

Comment:
It seems inconsistent that some default String value be defined as null while some others as an empty string. Could we settle on just a single way of doing things?